semgrep_output_v1.atd

```diff
@@ -1104,6 +1104,8 @@ type found_dependency = {
     <ts repr="map">;
   ?resolved_url: string option;
   transitivity: transitivity;
+  (* lockfile_path and line_number point to some location in some file *)
```

Comment:
add since semgrep 1.87.0 in comment

Reply:
Added in [1a14725](https://github.com/semgrep/semgrep-interfaces/pull/289/commits/1a14725f4eef424be7b011510b68b17ed6f80787)!